Cauldron/Assets/Scripts/UI/Menu/MainMenu.cs

```diff
@@ -70,6 +71,7 @@ public void ResetGameData(bool saveLanguage = true)
             recipeProvider.Reset();
             endingsProvider.Reset();
             milestoneProvider.Reset();
+            visitorsProvider.Reset();
```

Comment:
You can remove this Reset, I now clean all the files from the folder here)
And then you won't need this service in the project scope.
Good thing you thought about it, though.